src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,205 @@
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String insertCarRequest = "INSERT INTO cars(model, manufacturer_id) VALUES(?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createCarStatement =
+                        connection.prepareStatement(
+                                insertCarRequest, Statement.RETURN_GENERATED_KEYS)) {
+            createCarStatement.setString(1, car.getModel());
+            createCarStatement.setObject(2, car.getManufacturer().getId());
+            createCarStatement.executeUpdate();
+            ResultSet generatedKeys = createCarStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't insert car " + car + " into DB", e);
+        }
+        insertDriversOfTheCar(car);
+        return car;
+    }
+
+    private void insertDriversOfTheCar(Car car) {
+        String insertDriversRequest =
+                "INSERT INTO cars_drivers (driver_id, car_id) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement insertDriverStatement =
+                        connection.prepareStatement(insertDriversRequest)) {
+            insertDriverStatement.setLong(2, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                insertDriverStatement.setLong(1, driver.getId());
+                insertDriverStatement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Couldn't insert driver for the car" + car, e);
+        }
+
+    }
+
+    @Override
+    public Car get(Long id) {
+        Car car = null;
+        String getCarByIdRequest = "SELECT c.id, c.model, c.manufacturer_id, m.name, m.country \n"
+                + "FROM cars c JOIN manufacturers m\n"
+                + "ON c.manufacturer_id = m.id\n"
+                + "WHERE c.id = ? AND c.is_deleted = FALSE;";
+        try (Connection connection =
+                     ConnectionUtil.getConnection();
+                PreparedStatement getCarStatement =
+                        connection.prepareStatement(getCarByIdRequest)) {
+            getCarStatement.setLong(1, id);
+            ResultSet resultSet = getCarStatement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCarFromResultSetWithManufacturer(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get car from DB with id " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getDriversForCar(id));
+        }
+        return car;
+    }
+
+    private Car parseCarFromResultSetWithManufacturer(ResultSet resultSet) throws SQLException {
```

Comment:
```suggestion
    private Car parseCarWithManufacturer(ResultSet resultSet) throws SQLException {
```
same explanation as for driver